e2e/tests/room/11-autoTranslate.spec.ts

```diff
@@ -171,35 +171,35 @@ describe('Auto Translate', () => {
 
 	it('should see new message translated', async () => {
 		const randomMatcher = random();
-		const data = await sendMessageOnTranslationTestRoom(`${newMessage[languages.default]} - ${randomMatcher}`);
-		await searchMessage(`${newMessage[languages.default]} - ${randomMatcher}`, textMatcher); // will scroll the messages list to the last one
-		await waitForVisibleTextMatcher(`${newMessage[languages.translated]} - ${randomMatcher}`, textMatcher);
+		const data = await sendMessageOnTranslationTestRoom(`${newMessage[languages.default]}. Rc${randomMatcher}`);
```

Comment:
Correct me if I'm wrong, but if you're creating a new room for this test, you don't need a random string on each message. That makes it easier to translate the content.

Reply:
We aren't creating a new room for this. We are using the same room for that.
So, if we don't use the "random" content, then run multiple tests in parallel one will interfere with another because the label will be the same